workspaces/acs/.changeset/sixty-banks-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-acs': major
```

Comment:
Are you sure you want this to ship as `1.0.0` to start? My recommendation is to start at `0.1.0` that way you can get feedback from early adopters before pushing out your first major version. If you want to do that then set this to `patch`.

It's your plugin but just like to clarify this in case that's not what you wanted 👍

Reply:
Oh sure, no I'll fix it. Thanks for your recommendation here.